cypress/integration/podcasts.spec.js

```diff
@@ -6,14 +6,27 @@ describe('podcasts', () => {
   });
 
   it('renders many podcast cards', () => {
-    cy.get('[data-testid="podcast-card"]').should('have.length.greaterThan', 30);
+    cy.get('[data-testid="Card"]').should('have.length.greaterThan', 30);
   });
 
-  it('checks if audio is playing', () => {
-    cy.audio('episode.source').should('be.playing', true);
+  it('renders "Podcast Episodes" after filtering for Card', () => {
+    cy.get('[data-testid="Card"]')
+      .type('podcast-card', { force: true })
+      .type('{enter}');
+
+    cy.get('[data-testid="Card" podcast-card]').should('exist');
```

Comment:
How are you imagining this selector would work?

Reply:
Well I was noticing that when having `'[data-testid="podcast-card"` I was getting an error saying that data-testid couldn't be found so my thought process was chaining it. Finding the parent element Card first and then from there finding podcast-card.

Reply:
Does this selector work as written?

Reply:
No this is passed off of CircleCI on the PR

Reply:
You don't need to test in CI. Try running Cypress locally!

`yarn dev` in one shell and `yarn test:e2e` in another